packages/expo/bin/cli.ts

```diff
@@ -0,0 +1,74 @@
+#!/usr/bin/env node
+import arg from 'arg';
+import chalk from 'chalk';
+
+const defaultCmd = 'config';
+
+export type Command = (argv?: string[]) => void;
+
+const commands: { [command: string]: () => Promise<Command> } = {
+  // Add a new command here
+  config: () => import('../cli/config').then((i) => i.expoConfig),
+};
+
+const args = arg(
+  {
+    // Types
+    '--version': Boolean,
+    '--help': Boolean,
+
+    // Aliases
+    '-v': '--version',
+    '-h': '--help',
+  },
+  {
+    permissive: true,
+  }
+);
+
+if (args['--version']) {
+  // Version is added in the build script.
+  console.log(process.env.__EXPO_VERSION);
+  process.exit(0);
+}
+
+// Check if we are running `npx expo <subcommand>` or `npx expo`
+const isSubcommand = Boolean(commands[args._[0]]);
+
+// Handle `--help` flag
+if (!isSubcommand && args['--help']) {
+  console.log(chalk`
```

Comment:
Any particular reason why this file doesn't use `Log.log`?

Reply:
bin and cli are two separate builds so I'm attempting to keep them divided. The `Log.log` function exists mostly for testing.